### PR TITLE
💄feat(profile): SKFP-471 add user profile view

### PR DESCRIPTION
**Pull request by lflangis**
# FEATURE

- closes [#471](https://d3b.atlassian.net/browse/SKFP-471)

## Description
The Profile view page that is currently in KF does not fit with the FerlabUI components. We will also simplify the fields that are asked to be filled out by the user. The Profile page layout will be the same as in INCLUDE but the fields will differ slightly. The Profile View page will display the information that was filled in by the user in the Profile Edit page. The specifications are indicated in the notion page below.

Add a new Header background image

Add alert banner if a user has their profile as private. Clicking on “settings” will redirect the user to the Profile Edit page
Acceptance Criterias

## Screenshot (Before and After)

Profile Header
country_only
no_country_or_state
state_only

Community navigation (values are mockeD)
https://user-images.githubusercontent.com/65532894/194068069-e88343da-2646-407e-a789-3fee76e83b34.mp4

Live edit
https://user-images.githubusercontent.com/65532894/194067250-838b3bc6-b983-48da-84f1-4732ee31c701.mp4

Private profile alert
https://user-images.githubusercontent.com/65532894/194067352-90d9eaa0-6015-444c-9a8c-869b169a5670.mp4

## KNOW ISSUE

Diseases and studies interests containt duplicated values. Since there are all saved in the same fields `interests` in persona, it will create a duplicate in both fields. A ticket has been created to fix this issue later (we will change the interests to be all unique).

https://user-images.githubusercontent.com/65532894/194067075-659863d0-7066-4817-ac0f-208744a1f04e.mp4

### CODE REVIEW

File: src/components/Layout/Header/index.tsx [hunk 1]
```diff
@@ -228,6 +228,7 @@ const Header = () => {
                     onClick: () => {
                       dispatch(personaActions.cleanLogout());
                       dispatch(userActions.cleanLogout());
+                      window.sessionStorage.clear();
```
Comment (evans-g-crsj) [inline src/components/Layout/Header/index.tsx:231]: I think it is good but did you check with the product owner is that behaviour is ok (i think we ll lost the querybuilder stuff)?

File: src/store/persona/slice.ts [hunk 2]
```diff
@@ -15,11 +21,22 @@ const personaSlice = createSlice({
   name: 'persona',
   initialState: PersonaState,
   reducers: {
-    cleanLogout: (_) => PersonaState,
+    cleanLogout: (state) => {
+      state.profile = undefined;
```
Comment (evans-g-crsj) [inline src/store/persona/slice.ts:25]: I think that `null` is best suited. it is up to you.

File: src/views/Community/Member/index.tsx [hunk 3]
```diff
@@ -79,28 +120,71 @@ const CommunityMember = () => {
                       }}
                     />
                   </Col>
-                  <Col span={24}>
-                    <Typography.Title level={5}>
-                      {intl.get('screen.memberProfile.usageTitle')}
-                    </Typography.Title>
-                  </Col>
+                  {(!!diseasesInterest?.length || !!studiesInterest?.length) && (
```
Comment (evans-g-crsj) [inline src/views/Community/Member/index.tsx:123]: I think it s best to use `> 0` vs `!!`

File: src/views/Community/components/MemberCard/index.tsx [hunk 4]
```diff
@@ -15,26 +14,21 @@ interface OwnProps {
...
-        )}
-      </Space>
-    </Card>
-    // </Link>
+    <Link key={user._id} className={styles.memberLink} to={`/member/${user._id}`}>
+      <Card className={styles.memberCard}>
+        <Space direction="vertical" align="center">
+          <Gravatar className={styles.userGravatar} circle email={user.email || ''} />
+          <Typography.Title className={styles.memberCardName} level={5}>
+            <Highlighter textToHighlight={formatName(user) ?? ''} searchWords={[match]} />
```
Comment (evans-g-crsj) [inline src/views/Community/components/MemberCard/index.tsx:22]: maybe `formatName` could handle the fallback value `''`

**Review by evans-g-crsj — APPROVED** [review]

File: src/components/Layout/Header/index.tsx (hunk 1 above)
Comment (lflangis) [reply src/components/Layout/Header/index.tsx:231]: query use the localstorage and not the sessionstorage, should have no impact :)

File: src/store/persona/slice.ts (hunk 2 above)
Comment (lflangis) [reply src/store/persona/slice.ts:25]: I think `null` should not be used and `undefined` should priorized https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined

File: src/views/Community/components/MemberCard/index.tsx (hunk 4 above)
Comment (lflangis) [reply src/views/Community/components/MemberCard/index.tsx:22]: `  user.firstName && user.lastName ? `${user.firstName} ${user.lastName}` : user.email;` seems like it need to return the email by default, so maybe review it later when removing persona

**Pull request merged**